### PR TITLE
Added blog posts for community calls

### PR DESCRIPTION
**Pull request by JeanRoca**
# Description

Added blog posts for community calls after March. 

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/7970)

### CODE REVIEW

**Review by jwoo-msft — APPROVED** [review]

File: source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-April.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: 📣 Community Call - April, 2022
+subtitle: Overview on Universal Actions + Messaging Extensions
+date: 2022-04-14
+featured_image: https://pnp.github.io/blog/adaptive-cards-community-call/adaptive-cards-community-call-april-2022/images/recording-adaptive-cards-april-call_hua50012fa682050558ff5eb743a07f983_295297_700x0_resize_q100_h2_box_3.webp
+github_username: JeanRoca
+twitter: jpthepm
+---
+
+The following topics are covered in this month’s community call. Overview on Universal Actions + Messaging Extensions (elevating user experiences with Universal Actions and Messaging Extensions on Teams and Outlook – walkthrough capabilities and value propositions) - Karan Thapar (Microsoft), Accessibility Updates (Accessibility work and SDK v1.6 release) – [J.P. Roca](http://twitter.com/jpthepm) (Microsoft), and 15 minute live Q&A primarily focused on Universal Actions + Messaging Extensions. This call was hosted by [J.P. Roca](http://twitter.com/jpthepm) (Microsoft) \| @jpthepm. Recorded on April 14, 2022.
```
Comment (beervoley) [inline source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-April.md:10]:
```suggestion
The following topics are covered in this month’s community call. Overview on Universal Actions + Messaging Extensions (elevating user experiences with Universal Actions and Messaging Extensions on Teams and Outlook – walkthrough capabilities and value propositions) - Karan Thapar (Microsoft), Accessibility Updates (Accessibility work and SDK v1.6 release) – [J.P. Roca](http://twitter.com/jpthepm) (Microsoft), and 15 minutes live Q&A primarily focused on Universal Actions + Messaging Extensions. This call was hosted by [J.P. Roca](http://twitter.com/jpthepm) (Microsoft) \| @jpthepm. Recorded on April 14, 2022.
```

File: source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-April.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+### Agenda:
+- Universal Actions + Messaging Extensions - Karan Thapar (Microsoft) – [00:37](https://youtu.be/6Z8U4bSicgk?t=37)
+- Accessibility Updates - [J.P. Roca](http://twitter.com/jpthepm) (Microsoft) \| @jpthepm – [21:02](https://youtu.be/6Z8U4bSicgk?t=1262)
+- Q&A - J.P. Roca (Microsoft) \| @jpthepm & [Karan Thapar](https://www.linkedin.com/in/karanthapar91/) (Microsoft) –[22:04](https://youtu.be/6Z8U4bSicgk?t=1324)
+
+### Demo:
+**Elevate user experiences with Universal Actions and Messaging Extensions on Teams and Outlook** – Adaptive Cards enable you to take quick actions right from within Teams and Outlook. Hear the latest on Universal Actions, vacation approval scenario with value propositins, contextual or role-based views on Teams, sequential workflows on Teams, messaging extensions, search (read only) messaging extensions, action messaging extensions, link unfurling, success stories and value, how to onboard and build resources and Q&A at end.
```
Comment (beervoley) [inline source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-April.md:22]:
```suggestion
**Elevate user experiences with Universal Actions and Messaging Extensions on Teams and Outlook** – Adaptive Cards enable you to take quick actions right from within Teams and Outlook. Hear the latest on Universal Actions, vacation approval scenarios with value propositins, contextual or role-based views on Teams, sequential workflows on Teams, messaging extensions, search (read-only) messaging extensions, action messaging extensions, link unfurling, success stories, and value, how to onboard and build resources and Q&A at the end.
```

File: source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-April.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+* Demo – [Adaptive Cards Video Player Bot on Teams](https://youtu.be/-Y7_nNwC20w?t=28) - [Alexis Kinzelin](https://www.linkedin.com/in/alexis-kinzelin-214b6b20/) (Microsoft)
+* Tool - [Microsoft Graph Toolkit](https://docs.microsoft.com/graph/toolkit/overview)
+
+### Resources in General: 
+- Let us know the features you need    https://aka.ms/ACRoadmap
+- Schema Explorer - https://adaptivecards.io/explorer/Action.Execute.html
+- Designer - https://adaptivecards.io/designer/ 
+- Get started with Templating - https://aka.ms/ACTemplating
+- Browse the Adaptive Cards Code - https://aka.ms/ACRepo
+- Find tools, sample cards and more - https://www.madewithcards.io
```
Comment (beervoley) [inline source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-April.md:38]:
```suggestion
- Find tools, sample cards, and more - https://www.madewithcards.io
```

File: source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-August.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+---
+title: 📣 Community Call - August, 2022
+subtitle: Overview of the React Native Adaptive Cards Renderer by BigThinkCode
+date: 2022-08-11
+featured_image: https://pnp.github.io/blog/adaptive-cards-community-call/adaptive-cards-community-call-august-2022/images/recording-adaptive-cards-august-call_hu29a577d313165ec630d79518bb1a376e_280808_700x0_resize_q100_h2_box_3.webp
+github_username: JeanRoca
+twitter: jpthepm
+---
+
+In this month’s community call, get an overview of the React Native Adaptive Cards Renderer and on usage of the React Native Adaptive Cards SDK in a Microsoft Viva Connections mobile app and in a LUIS chatbot app delivered by CEO & Founder – [Vasanthakumar Sarpasayanam](https://in.linkedin.com/in/vasanthakumar-sarpasayanam) and his team at BigThinkCode. The community-supported React Native renderer for Adaptive Cards is maintained by BigThinkCode. There was a 10-minute Q&A covering a host of topics. This call was hosted by [J.P. Roca](http://twitter.com/jpthepm) (Microsoft) \| @jpthepm. Recorded on August 11, 2022.
```
Comment (beervoley) [inline source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-August.md:10]:
```suggestion
In this month’s community call, get an overview of the React Native Adaptive Cards Renderer and on the usage of the React Native Adaptive Cards SDK in a Microsoft Viva Connections mobile app and in a LUIS chatbot app delivered by CEO & Founder – [Vasanthakumar Sarpasayanam](https://in.linkedin.com/in/vasanthakumar-sarpasayanam) and his team at BigThinkCode. The community-supported React Native renderer for Adaptive Cards is maintained by BigThinkCode. There was a 10-minute Q&A covering a host of topics. This call was hosted by [J.P. Roca](http://twitter.com/jpthepm) (Microsoft) \| @jpthepm. Recorded on August 11, 2022.
```

File: source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-August.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    * Documentation - [Overview of Viva Connections Extensibility](https://docs.microsoft.com/sharepoint/dev/spfx/viva/overview-viva-connections)
+    * Extensions - [Adaptive Card Extension Design Gallery](https://appsource.microsoft.com/product/office/WA200003929)
+
+### Resources in General: 
+- Let us know the features you need    https://aka.ms/ACRoadmap
+- Schema Explorer - https://adaptivecards.io/explorer/Action.Execute.html
+- Designer - https://adaptivecards.io/designer/ 
+- Get started with Templating - https://aka.ms/ACTemplating
+- Browse the Adaptive Cards Code - https://aka.ms/ACRepo
+- Find tools, sample cards and more - https://www.madewithcards.io
```
Comment (beervoley) [inline source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-August.md:58]:
```suggestion
- Find tools, sample cards, and more - https://www.madewithcards.io
```

File: source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-July.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
+---
+title: 📣 Community Call - July, 2022
+subtitle: Action.ToggleVisibility Deprecation (non-breaking change)
+date: 2022-07-14
+featured_image: https://pnp.github.io/blog/adaptive-cards-community-call/adaptive-cards-community-call-july-2022/images/recording-adaptive-cards-july-call_hufb42edcfeb6bfc1cc800f7fd057797ff_271174_700x0_resize_q100_h2_box_3.webp
+github_username: JeanRoca
+twitter: jpthepm
+---
+
+In this month’s community call the following topics were presented - Action.ToggleVisibility Deprecation (a non-breaking change for a control that needs to support accessibility requirements for all users), GA release for Adaptive Cards Handle Responses Connector, teaser on the React Native Adaptive Cards Renderer capability used in Microsoft Viva Connections mobile app to be demoed next month, and a 10-minute Q&A covering a host of topics - release schedule, integration, challenges and demo requests. This call was hosted by [J.P. Roca](http://twitter.com/jpthepm) (Microsoft) \| @jpthepm. Recorded on July 14, 2022.
```
Comment (beervoley) [inline source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-July.md:10]:
```suggestion
In this month’s community call the following topics were presented - Action.ToggleVisibility Deprecation (a non-breaking change for a control that needs to support accessibility requirements for all users), GA release for Adaptive Cards Handle Responses Connector, teaser on the React Native Adaptive Cards Renderer capability used in Microsoft Viva Connections mobile app to be demoed next month, and a 10-minute Q&A covering a host of topics - release schedule, integration, challenges, and demo requests. This call was hosted by [J.P. Roca](http://twitter.com/jpthepm) (Microsoft) \| @jpthepm. Recorded on July 14, 2022.
```

File: source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-July.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+    * Microsoft Build Presentation: [Building great apps with the open platform of Windows (microsoft.com)](https://mybuild.microsoft.com/en-US/sessions/3a0b225f-279e-4973-8106-20ce58f16ed1?source=/home) (Windows 11 Widgets - a.k.a., Adaptive Cards at 40:56)
+    * Feedback Portal – [Microsoft Viva Connections](https://feedbackportal.microsoft.com/feedback/forum/5092ec4b-1d1c-ec11-b6e7-0022481f8472)
+
+### Resources in General: 
+- Let us know the features you need    https://aka.ms/ACRoadmap
+- Schema Explorer - https://adaptivecards.io/explorer/Action.Execute.html
+- Designer - https://adaptivecards.io/designer/ 
+- Get started with Templating - https://aka.ms/ACTemplating
+- Browse the Adaptive Cards Code - https://aka.ms/ACRepo
+- Find tools, sample cards and more - https://www.madewithcards.io
```
Comment (beervoley) [inline source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-July.md:52]:
```suggestion
- Find tools, sample cards, and more - https://www.madewithcards.io
```

File: source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-June.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
+---
+title: 📣 Community Call - June, 2022
+subtitle: New Teams Toolkit v4.0 Adaptive Cards features
+date: 2022-06-09
+featured_image: https://pnp.github.io/blog/adaptive-cards-community-call/adaptive-cards-community-call-june-2022/images/recording-adaptive-cards-june-call_hu262cf22b54ffadc2b9a48f80d7c7bfec_263625_700x0_resize_q100_h2_box_3.webp
+github_username: JeanRoca
+twitter: jpthepm
+---
+
+In this month’s community call the focus is on the newly released Teams Toolkit v4.0. This latest Teams VS Code extension enables developers/non-developers to create Teams apps with minimal work. v4 capabilities called out include scenario-based app creation, run your Teams apps on Office and Outlook, create an app from sample, select to uncomment/activate commented out capabilities and seeing your new app in Run and Debug mode. With the Adaptive Cards VS Code extension installed, see side-by-site edit-render experience similar to Adaptive Card Designer. The 20-minute Q&A covers a host of topics. This call was hosted by [J.P. Roca](http://twitter.com/jpthepm) (Microsoft) \| @jpthepm. Recorded on June 9, 2022.
```
Comment (beervoley) [inline source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-June.md:10]: I'd rather make all those verbs gerund tho:)
```suggestion
In this month’s community call the focus is on the newly released Teams Toolkit v4.0. This latest Teams VS Code extension enables developers/non-developers to create Teams apps with minimal work. v4 capabilities called out include scenario-based app creation, run your Teams apps on Office and Outlook, create an app from sample, select to uncomment/activate commented out capabilities, and see your new app in Run and Debug mode. With the Adaptive Cards VS Code extension installed, see side-by-site edit-render experience similar to Adaptive Card Designer. The 20-minute Q&A covers a host of topics. This call was hosted by [J.P. Roca](http://twitter.com/jpthepm) (Microsoft) \| @jpthepm. Recorded on June 9, 2022.
```

File: source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-June.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+    * Documentation - [Build your first tab app using JavaScript](https://docs.microsoft.com/microsoftteams/platform/sbs-gs-javascript?tabs=vscode%2Cvsc%2Cviscode&%3FWT.mc_id=m365-55688-timura) \| aka.ms/TeamsAppJS
+    * Twitter - [Tomomi Imura](http://twitter.com/girlie_mac) \| @girlie_mac
+
+### Resources in General: 
+- Let us know the features you need    https://aka.ms/ACRoadmap
+- Schema Explorer - https://adaptivecards.io/explorer/Action.Execute.html
+- Designer - https://adaptivecards.io/designer/ 
+- Get started with Templating - https://aka.ms/ACTemplating
+- Browse the Adaptive Cards Code - https://aka.ms/ACRepo
+- Find tools, sample cards and more - https://www.madewithcards.io
```
Comment (beervoley) [inline source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-June.md:38]:
```suggestion
- Find tools, sample cards, and more - https://www.madewithcards.io
```

File: source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-June.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+- Contribution Guidelines – https://aka.ms/ACRepo 
+
+### Adaptive Cards feedback channels
+
+-   [Issues and feature requests](https://github.com/Microsoft/AdaptiveCards/issues)
+-   [Technical questions](https://stackoverflow.com/questions/tagged/adaptive-cards)
+
+The Adaptive Cards developer community call is on the 2nd Thursday of each month. We will provide you with the opportunity to learn about Adaptive Cards in general, how they could be a great tool in your applications and contribute to the evolution of the format and SDKs. Each month, we'll discuss new features we are bringing to the platform, our roadmap, and listen to your feedback and suggestions.
```
Comment (beervoley) [inline source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-June.md:46]:
```suggestion
The Adaptive Cards developer community call is on the 2nd Thursday of each month. We will provide you with the opportunity to learn about Adaptive Cards in general and show how they could be a great tool in your applications and contribute to the evolution of the format and SDKs. Each month, we'll discuss new features we are bringing to the platform, our roadmap, and listen to your feedback and suggestions.
```

File: source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-May.md [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+    * Article – [Microsoft Power Automate: Post Adaptive cards to a Microsoft Teams for an Approval](https://powerusers.microsoft.com/t5/Power-Automate-Community-Blog/Microsoft-Power-Automate-Post-Adaptive-cards-to-a-Microsoft/ba-p/658693)
+
+
+### Resources in General: 
+- Let us know the features you need    https://aka.ms/ACRoadmap
+- Schema Explorer - https://adaptivecards.io/explorer/Action.Execute.html
+- Designer - https://adaptivecards.io/designer/ 
+- Get started with Templating - https://aka.ms/ACTemplating
+- Browse the Adaptive Cards Code - https://aka.ms/ACRepo
+- Find tools, sample cards and more - https://www.madewithcards.io
```
Comment (beervoley) [inline source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-May.md:41]:
```suggestion
- Find tools, sample cards, and more - https://www.madewithcards.io
```

File: source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-October.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+* Documentation – [Widget provider package manifest XML format](https://learn.microsoft.com/windows/apps/develop/widgets/widget-provider-manifest)
+* Feedback – [Want to contribute to Microsoft Adaptive Cards?](https://github.com/microsoft/AdaptiveCards/issues)
+
+### Resources in General: 
+- Let us know the features you need    https://aka.ms/ACRoadmap
+- Schema Explorer - https://adaptivecards.io/explorer/Action.Execute.html
+- Designer - https://adaptivecards.io/designer/ 
+- Get started with Templating - https://aka.ms/ACTemplating
+- Browse the Adaptive Cards Code - https://aka.ms/ACRepo
+- Find tools, sample cards and more - https://www.madewithcards.io
```
Comment (beervoley) [inline source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-October.md:43]:
```suggestion
- Find tools, sample cards, and more - https://www.madewithcards.io
```

**Review by beervoley — APPROVED** [review]

File: source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-April.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+
+### Agenda:
+- Universal Actions + Messaging Extensions - Karan Thapar (Microsoft) – [00:37](https://youtu.be/6Z8U4bSicgk?t=37)
+- Accessibility Updates - [J.P. Roca](http://twitter.com/jpthepm) (Microsoft) \| @jpthepm – [21:02](https://youtu.be/6Z8U4bSicgk?t=1262)
+- Q&A - J.P. Roca (Microsoft) \| @jpthepm & [Karan Thapar](https://www.linkedin.com/in/karanthapar91/) (Microsoft) –[22:04](https://youtu.be/6Z8U4bSicgk?t=1324)
+
+### Demo:
+**Elevate user experiences with Universal Actions and Messaging Extensions on Teams and Outlook** – Adaptive Cards enable you to take quick actions right from within Teams and Outlook. Hear the latest on Universal Actions, vacation approval scenarios with value propositins, contextual or role-based views on Teams, sequential workflows on Teams, messaging extensions, search (read-only) messaging extensions, action messaging extensions, link unfurling, success stories, and value, how to onboard and build resources and Q&A at the end.
```
Comment (JeanRoca) [inline source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-April.md:22]:
``` suggestion
**Elevate user experiences with Universal Actions and Messaging Extensions on Teams and Outlook** – Adaptive Cards enable you to take quick actions right from within Teams and Outlook. Hear the latest on Universal Actions, vacation approval scenarios with value propositions, contextual or role-based views on Teams, sequential workflows on Teams, messaging extensions, search (read-only) messaging extensions, action messaging extensions, link unfurling, success stories, and value, how to onboard and build resources and Q&A at the end.
```

**Pull request merged**